persistence-utils-demo/pom.xml

```diff
@@ -23,7 +23,7 @@
       <plugin>
```

Comment:
the dependency to 'persistenc-utils' itself is still referenced as IAR ... so it will not be inclued -> switch to JAR or skip the type altogether and we are fine.